### PR TITLE
hotfix/22.5.4/add new logging format

### PR DESCRIPTION
**Pull request by CeEv**
hotfix 22.5.4

# Checkliste

- [x] WIP PR-Label entfernt, wenn die Checkliste abgearbeitet wurde

## Mehr
Weitere Informationen zur DoD [hier im Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5209477&bid=17083947)

File: config/global.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ const exp = {
 	MAXIMUM_ALLOWABLE_TOTAL_ATTACHMENTS_SIZE_BYTE,
 };
 
-logger.info(util.inspect(exp, { depth: 1, compact: false }));
+// eslint-disable-next-line no-console
+console.log(util.inspect(exp, { depth: 1, compact: false }));
```
Comment (janrenz) [inline config/global.js]: console?

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5209477&bid=17083947)

**Comment by adrianjost** [comment]
@CeEv still `do_not_merge`?

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5209477&bid=17083947)

**Comment by CeEv** [comment]
> 
> 
> @CeEv still `do_not_merge`?

Die hotfixes werden doch immer von lokal direkt auf master/development gepushed.

File: config/global.js (hunk 1 above)
Comment (CeEv) [reply config/global.js]: > 
> 
> version update in der package-lock fehlt

done

**Comment by CeEv** [comment]
> 
> 
> version update in der package-lock fehlt

done

**Comment by adrianjost** [comment]
> > @CeEv still `do_not_merge`?
> 
> Die hotfixes werden doch immer von lokal direkt auf master/development gepushed.

man kann es auch per merges machen

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5209477&bid=17083947)

**Pull request merged**